src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double upperA; //Length of the upper base
+    private final double lowerB; // Length of the lower base
+    private final double height; // height
+
+    public IsoscelesTrapezoid(double upperA, double lowerB, double height) {
+        this.upperA = upperA;
+        this.lowerB = lowerB;
+        this.height = height;
+        setColor(new ColorSupplier().getRandomColor());
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("Figure: isoscales trapezoid, area: " + obtainTheArea()
```

Comment:
There's a typo in the word 'isoscales'. It should be 'isosceles'. Also, use 'area' instead of 'obtainTheArea' to fix the issue with the method name as mentioned in another comment.